system-variables.md

```diff
@@ -1655,9 +1655,9 @@ SET tidb_query_log_max_len = 20
 ### tidb_restricted_read_only <span class="version-mark">New in v5.2.0</span>
 
 - Scope: GLOBAL
-- Default value: `0`
-- Value options: `0`, `1`
-- This variable controls the read-only status of the entire cluster. If the variable is enabled (which means that the value is `1`), all TiDB servers in the entire cluster are in the read-only mode. In this case, TiDB only executes the statements that do not modify data, such as `SELECT`, `USE`, and `SHOW`. For other statements such as `INSERT` and `UPDATE`, TiDB rejects executing those statements in the read-only mode.
+- Persists to cluster: Yes
```

Comment:
Is this "Persists to cluster: Yes" setting already used all the way back to v5.2.x?

Reply:
Let's not cherry pick this part. It technically does persist to cluster, but what that means is only explained in the 6.1 docs.

Reply:
OK